VERSION

```diff
@@ -1 +1 @@
-0.2.1
+0.3.0-dev
```

Comment:
As this is a backwards-compatible fix, I believe a 0.2.2 version should be sufficient.